src/ui/sass/modules/_DirectAnswer.scss

```diff
@@ -81,7 +81,7 @@ $direct-answer-footer-height: var(--yxt-module-footer-height) !default;
   {
     display: flex;
     margin-right: calc(var(--yxt-base-spacing) / 2);
-    height: 18px;
+    color: inherit;
```

Comment:
Could we make it so this styling is only on `yxt-DirectAnswer-titleIconWrapper.yxt-Results-titleIconWrapper`?

Reply:
sure, updated!